packages/zwave-js/src/lib/commandclass/BarrierOperatorCC.ts

```diff
@@ -105,8 +112,7 @@ export class BarrierOperatorCCAPI extends CCAPI {
 			this.commandOptions,
 		))!;
 		return {
-			// interpret unknown values as false
-			currentValue: response.currentValue,
+			state: response.state,
```

Comment:
`BarrierOperatorCCReport` has two properties that work hand in hand - so this should return both.

Reply:
you right!